### ISSUE TITLE
Windows 10 ofxNatNet: Net Exception

### ISSUE THREAD

**Issue by ikbenmacje** [hku-ect/NatNet2OSCbridge#10]
This issue is related to ofxNatnet and not NatNet2OSCbridge see the issue [here](https://github.com/satoruhiga/ofxNatNet/issues/9)

**WORKAROUND**
To get around this problem disable IPV6 in your netwrk adapter settings.
See attached image:

![networkadaptersettings](https://user-images.githubusercontent.com/1862254/31181038-975ee044-a920-11e7-8fc2-c57c30156865.png)

**Comment by tonyschuite** [comment]
still an issue. IPV6 solution not working.

**Comment by sphaero** [comment]
console will print:
```
ofxNatNet: I/O  error
```
Solution is to disable ipv6 in windows

**Comment by sphaero** [comment]
@tonyschuite could you elaborate or at least confirm you still have the issue?

latest nightly can be found at http://pong.hku.nl/~buildbot/nightly

**Comment by tonyschuite** [comment]
how would you like me to elaborate.
Mocap laptop has ipv6 turned off, still same error in console and no data / connection.

**Comment by sphaero** [comment]
Yes, please try the nightly and also specify which nightly and which windows version

**Comment by GordeyChernyy** [comment]
It worked for me
